code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -637,12 +640,14 @@
 	armor = list("melee" = 20, "bullet" = 15, "laser" = 15, "energy" = 45, "bomb" = 100, "bio" = 100, "rad" = 100, "fire" = 100, "acid" = 100)
 	item_color = "ancient"
 	brightness_on = 16
-	clothing_flags = STOPSPRESSUREDAMAGE | THICKMATERIAL | BLOCK_GAS_SMOKE_EFFECT | ALLOWINTERNALS | SCAN_REAGENTS
 	flash_protect = 5 //We will not be flash by bombs
 	tint = 1
 	var/obj/machinery/doppler_array/integrated/bomb_radar
 	max_heat_protection_temperature = FIRE_IMMUNITY_MAX_TEMP_PROTECT
+	clothing_flags = STOPSPRESSUREDAMAGE | THICKMATERIAL | BLOCK_GAS_SMOKE_EFFECT | ALLOWINTERNALS | SCAN_REAGENTS | IMMUTABLE_SLOW
```

Comment:
the mason rig helmet doesn't slowdown, why would it need the `IMMUTABLE_SLOW` flag?

Reply:
So people dont use a speed up on the hat

Reply:
Speed potions just zero the speed modifier! You should know! The speed modifier is already 0 here!